Makefile

```
@@ -247,11 +247,11 @@ E2E_TIMEOUT ?= 90m
 E2E_TEST_NS ?= operators
 E2E_INSTALL_NS ?= operator-lifecycle-manager
 E2E_CATALOG_NS ?= $(E2E_INSTALL_NS)
-GINKGO_OPTS += -v -randomize-suites -race -trace $(if $(E2E_FLAKE_ATTEMPTS),--flake-attempts='$(E2E_FLAKE_ATTEMPTS)') $(if $(E2E_SEED),-seed '$(E2E_SEED)') $(if $(TEST),-focus '$(TEST)',) $(if $(SKIP), -skip '$(SKIP)')
+E2E_GINKGO_OPTS := $(GINKGO_OPTS) -v -randomize-suites -race -trace $(if $(E2E_FLAKE_ATTEMPTS),--flake-attempts='$(E2E_FLAKE_ATTEMPTS)') $(if $(E2E_SEED),-seed '$(E2E_SEED)') $(if $(TEST),-focus '$(TEST)',) $(if $(SKIP), -skip '$(SKIP)')
 
 .PHONY: e2e
 e2e: #HELP Run e2e tests against a cluster running OLM (params: $E2E_TEST_NS (operator), $E2E_INSTALL_NS (operator-lifecycle-manager), $E2E_CATALOG_NS (operator-lifecycle-manager), $E2E_TIMEOUT (90m), $E2E_FLAKE_ATTEMPTS (1), $TEST(undefined))
-	$(GO_TEST_ENV) $(GINKGO) -timeout $(E2E_TIMEOUT) $(GINKGO_OPTS) ./test/e2e -- -namespace=$(E2E_TEST_NS) -olmNamespace=$(E2E_INSTALL_NS) -catalogNamespace=$(E2E_CATALOG_NS) $(E2E_OPTS)
+	$(GO_TEST_ENV) $(GINKGO) -timeout $(E2E_TIMEOUT) $(E2E_GINKGO_OPTS) ./test/e2e -- -namespace=$(E2E_TEST_NS) -olmNamespace=$(E2E_INSTALL_NS) -catalogNamespace=$(E2E_CATALOG_NS) $(E2E_OPTS)
```

So if someone had something using  `GINKGO_OPTS` they're now out of luck? They'd need to change to calling whatever they had to instead define E2E_GINKGO_OPTS? if you did the following, wouldn't it be backwards compatible:
```suggestion
	$(GO_TEST_ENV) $(GINKGO) -timeout $(E2E_TIMEOUT) $(E2E_GINKGO_OPTS) $(GINKGO_OPTS) ./test/e2e -- -namespace=$(E2E_TEST_NS) -olmNamespace=$(E2E_INSTALL_NS) -catalogNamespace=$(E2E_CATALOG_NS) $(E2E_OPTS)
```
That's what I thought the PR meant by splitting, as opposed to replacing.

---

hmm, well they can kinda still use the GINGKO_OPTS, it's pulled into E2E_GINKGO_OPTS, but whereas before they could define whatever they needed for GINGKO_OPTS, with this change they'd get a bunch of other stuff no matter what. It's not only somewhat backwards compatible without the suggestion I think.

---

Missed part of your comment. This was added a couple of weeks ago. The only people using it is us. But that's fair.

---

👍